lib/src/model/class.dart

```diff
@@ -85,6 +85,10 @@ class Class extends Container
   Iterable<Operator> get instanceOperators =>
       quiver.concat([super.instanceOperators, inheritedOperators]);
 
+  @override
+  bool get publicInheritedInstanceOperators =>
+      instanceOperators.every((f) => f.isInherited);
```

Comment:
Good catch.  I don't think members of `instanceOperators` are guaranteed to be public even if they happen to be now.   Wrapping in model_utils.filterNonPublic() seems prudent in case of changes.

These booleans really should be prefixed with "has" but given as this is consistent with methods, I'll save that adjustment for the next time through here.

Reply:
Seems we already had this with a `publicInstanceOperators` getter, so I switched to using that. Thanks for catching this.